### PR TITLE
Edits to improve English readability

### PR DESCRIPTION
**Pull request by chrisn**
***
<a href="https://pr-preview.s3.amazonaws.com/chrisn/edge-computing-web-exploration/pull/33.html" title="Last updated on Aug 18, 2023, 4:37 PM UTC (8fc2ffd)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/edge-computing-web-exploration/33/927dd2b...chrisn:8fc2ffd.html" title="Last updated on Aug 18, 2023, 4:37 PM UTC (8fc2ffd)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -304,63 +303,63 @@ <h2>Use Cases</h2>
...
+                Hybrid applications are applications whose main task includes both rendering intensive work and computing intensive work. For example, modern e-commerce mobile applications
                 leverage machine learning inference in the client side for AR/VR type user experiences. At the same time, e-commerce mobile applications need to fetch dynamic content based
                 on user preferences.
               </p>
         </dd>
         <dt>Mobile/static client:</dt><dd>
             <p>
-              Some client side applications remain static most of the time. For example, a camera for traffic monitoring and analysis does not require mobility support. 
-              On the other hand, some client side applications will change location continuously. For example, applications running on a connected vehicle or self driving vehicle 
+              Some client side applications remain static most of the time. For example, a camera for traffic monitoring and analysis does not require mobility support.
```
Comment (mmccool) [inline index.html]: Use of "static" here refers to location, not other forms of static (e.g. measurements not changing much).
```suggestion
              Some client side applications remain at a static location most of the time. For example, a camera for traffic monitoring and analysis does not require mobility support.
```

File: index.html [hunk 2]
```diff
@@ -304,63 +303,63 @@ <h2>Use Cases</h2>
...
+              On the other hand, some client side applications will change location continuously. For example, applications running on a connected vehicle or self driving vehicle
               will change location rapidly with the vehicle.
             </p>
         </dd>
        </dl>
 
 
       <p>
-        The use cases in the following sections are generally classified into the following categories based on the different work load type:
+        The use cases in the following sections are generally classified into the following categories based on the different workload type.
```
Comment (mmccool) [inline index.html]:
```suggestion
        The use cases in the following sections are generally classified into the following categories based on different workload types.
```

File: index.html [hunk 3]
```diff
@@ -841,8 +838,8 @@ <h3>Detailed Requirements</h3>
           <tbody>
             <tr>
               <td>Performance</td>
-              <td>R1: Client Offload.</td>
-              <td>Client should be able to offload computing intensive work 
+              <td>R1: Client Offload</td>
+              <td>Clients should be able to offload computing intensive work
                   to an edge resource.</td>
```
Comment (mmccool) [inline index.html]:
```suggestion
                  to an Edge Resource.</td>
```

File: index.html [hunk 4]
```diff
@@ -862,10 +859,10 @@ <h3>Detailed Requirements</h3>
             </tr>
             <tr>
               <td>Resiliency</td>
-              <td>R2a: Application-Directed Migration.</td>  
-              <td>The application should be able to explicitly manage migration 
+              <td>R2a: Application-Directed Migration</td>
+              <td>The application should be able to explicitly manage migration
                   of work between computing resources.
```
Comment (mmccool) [inline index.html]:
```suggestion
                  of work between Computing Resources.
```

File: index.html [hunk 5]
```diff
@@ -862,10 +859,10 @@ <h3>Detailed Requirements</h3>
...
-                  This may include temporarily running a workload on 
+                  This may include temporarily running a workload on
                   multiple computing resources to hide transfer latency.
```
Comment (mmccool) [inline index.html]:
```suggestion
                  multiple Computing Resources to hide transfer latency.
```

File: index.html [hunk 6]
```diff
@@ -886,12 +883,12 @@ <h3>Detailed Requirements</h3>
             </tr>
             <tr>
               <td>Resiliency</td>
-              <td>R2b: Live Migration.</td>
-              <td>The edge cloud should be able to transparently migrate 
+              <td>R2b: Live Migration</td>
+              <td>The edge cloud should be able to transparently migrate
```
Comment (mmccool) [inline index.html]:
```suggestion
              <td>The Edge Cloud should be able to transparently migrate
```

File: index.html [hunk 7]
```diff
@@ -886,12 +883,12 @@ <h3>Detailed Requirements</h3>
...
                 live (running) work between computing resources.
```
Comment (mmccool) [inline index.html]:
```suggestion
                live (running) work between Computing Resources.
```

File: index.html [hunk 8]
```diff
@@ -886,12 +883,12 @@ <h3>Detailed Requirements</h3>
...
-                This includes between edge resources, cloud resources, 
+                This includes between edge resources, cloud resources,
```
Comment (mmccool) [inline index.html]:
```suggestion
                This includes between Edge Resources, Cloud Resources,
```

File: index.html [hunk 9]
```diff
@@ -901,8 +898,8 @@ <h3>Detailed Requirements</h3>
             </tr>
             <tr>
               <td>Flexibility</td>
-              <td>R3: Discovery.</td>
-              <td>A client should be able to dynamically enumerate available 
+              <td>R3: Discovery</td>
+              <td>A client should be able to dynamically enumerate available
                   edge resources.</td>
```
Comment (mmccool) [inline index.html]:
```suggestion
                  Edge Resources.</td>
```

File: index.html [hunk 10]
```diff
@@ -922,10 +919,10 @@ <h3>Detailed Requirements</h3>
...
-              <td>R4: Selection.</td>
-              <td>A client should be able to select between available resources, 
+              <td>R4: Selection</td>
+              <td>A client should be able to select between available resources,
                   including making a decision about whether offload is appropriate
-                  (e.g. running on the client may be the best choice).
+                  (e.g., running on the client may be the best choice).
                   This selection may be automatic or application-directed,
                   and may require metadata or measurements
                   of the performance and latency of edge resources,
```
Comment (mmccool) [inline index.html]:
```suggestion
                  of the performance and latency of Edge Resources,
```

File: index.html [hunk 11]
```diff
@@ -952,10 +949,10 @@ <h3>Detailed Requirements</h3>
             </tr>
             <tr>
               <td>Flexibility</td>
-              <td>R5: Packaging.</td>
-              <td>A workload should be packaged so it can be executed on 
+              <td>R5: Packaging</td>
+              <td>A workload should be packaged so it can be executed on
                 a variety of edge resources.
```
Comment (mmccool) [inline index.html]:
```suggestion
                a variety of Edge Resources.
```

File: index.html [hunk 12]
```diff
@@ -1028,11 +1025,11 @@ <h3>Detailed Requirements</h3>
               <td>The client should be able to control
                   the use of resources by an
                   offloaded workload on a per-application basis.
-                Note: If an edge resource has a usage charge, for example, 
-                a client may want to set quotas on offload, 
+                Note: If an edge resource has a usage charge, for example,
```
Comment (mmccool) [inline index.html]:
```suggestion
                Note: If an Edge Resource has a usage charge, for example,
```

File: index.html [hunk 13]
```diff
@@ -1180,7 +1177,7 @@ <h3>Detailed Requirements</h3>
                   compute resources these need to be taken into account
```
Comment (mmccool) [inline index.html]:
```suggestion
                  Computing Resources these need to be taken into account
```

File: index.html [hunk 14]
```diff
@@ -1180,7 +1177,7 @@ <h3>Detailed Requirements</h3>
...
                   when estimating performance and selecting a compute
```
Comment (mmccool) [inline index.html]:
```suggestion
                  when estimating performance and selecting a Computing
```

File: index.html [hunk 15]
```diff
@@ -1180,7 +1177,7 @@ <h3>Detailed Requirements</h3>
...
                   resource to use for offload.  Access to such resources
```
Comment (mmccool) [inline index.html]:
```suggestion
                  Resource to use for offload.  Access to such resources
```

**Review by mmccool — APPROVED** [review]
Changes proposed by @chrisn are all good, I approve generally.  I have a few more suggestions but these are mostly addressing inconsistencies in capitalization of defined terms for Computing Resources, Edge Resources, etc. (which we could however deal with later; there are probably others...).

**Comment by chrisn** [comment]
Thanks. I've applied all your changes. I suggest we create a follow up PR for those additional edits.

**Pull request merged**